package.json

```diff
@@ -98,15 +100,15 @@
         "build": "xcodebuild -project ios/NativeSigner.xcodeproj -scheme NativeSigner -configuration Debug -sdk iphonesimulator -derivedDataPath ios/build/NativeSigner",
         "type": "ios.simulator",
         "device": {
-          "type": "iPhone SE"
+          "type": "iPhone 8"
```

Comment:
for backward compatible reason better to keep small screen iphone as tester phone

Reply:
iPhone 8 is the earliest simulator that comes with the latest Xcode tooling.